roofit/xroofit/src/xRooNode.cxx

```diff
@@ -8946,12 +8956,12 @@ void xRooNode::Draw(Option_t *opt)
          //                    int n = _size;
          //                    Int_t w = 1, h = 1;
          //                    if (pad->GetCanvas()->GetWindowWidth() > pad->GetCanvas()->GetWindowHeight()) {
-         //                        w = std::ceil(std::sqrt(n));
-         //                        h = std::floor(std::sqrt(n));
+         //                        w = TMath::Ceil(TMath::Sqrt(n));
+         //                        h = TMath::Floor(TMath::Sqrt(n));
```

Comment:
I was replacing many `TMath` calls in RooFit with `std::`functions automatically in RooFit recently. Sorry that I didn't notify you about this! Can the same change be made in xroofit?